data/mods/Aftershock/items/gun/laser.json

```
@@ -107,16 +107,175 @@
     "loudness": 8,
     "reload": 500,
     "valid_mod_locations": [ [ "accessories", 4 ], [ "sights", 1 ], [ "sling", 1 ], [ "stock", 1 ], [ "underbarrel", 1 ] ],
-    "ammo_effects": [ "LASER", "DRAW_AS_LINE" ],
-    "flags": [ "NEVER_JAMS", "NON_FOULING" ],
+    "ammo_effects": [ "LASER" ],
+    "flags": [ "NEVER_JAMS", "NON_FOULING", "NEEDS_NO_LUBE" ],
     "ammo_to_fire": 20,
     "pocket_data": [
       {
         "pocket_type": "MAGAZINE_WELL",
         "holster": true,
         "max_contains_volume": "20 L",
         "max_contains_weight": "20 kg",
-        "item_restriction": [ "heavy_battery_cell", "heavy_plus_battery_cell", "heavy_atomic_battery_cell", "heavy_disposable_cell" ]
+        "magazine_well": "75 ml",
+        "item_restriction": [ "afs_cartridge", "afs_bootleg_cartridge" ]
+      }
+    ]
+  },
+  {
+    "id": "afs_av22",
+    "type": "GUN",
+    "name": { "str": "Av-22" },
+    "copy-from": "v29",
+    "description": "A common pulse-laser pistol cleared for civilian use.  Recoiless, energy-efficient, and easy to use, the av-22 is often carried as defense weapon by those who lack the practice required to effectively use ballistic weaponry.  Due to its low power, it must be fired in sequence mode to stand a chance against hardened foes.",
```

```suggestion
    "description": "A common pulse-laser pistol cleared for civilian use.  Recoiless, energy-efficient, and easy to use, the av-22 is often carried as a defense weapon by those who lack the practice (or strength) required to effectively use ballistic weaponry.  Due to its low power, it must be fired in sequence mode to stand a chance against hardened foes.",
```
"sequence mode"? Burst?

---

Burst is called sequence in the UI